source/_integrations/media_player.markdown

```diff
@@ -156,6 +156,21 @@ Currently only supported on [Sonos](/integrations/sonos), [Spotify](/integration
 | `entity_id`            |      yes | Target a specific media player. For example `media_player.kitchen`|
 | `repeat`               |       no | `off`/`all`/`one` for setting repeat mode            |
 
+#### Service `media_player.join`
+
+Allows to group media players together for synchronous playback. Only works on supported multiroom audio systems.
+
+| Service data attribute | Optional | Description                                          |
+| ---------------------- | -------- | ---------------------------------------------------- |
+| `entity_id`            |       no | The media player entity whose playback will be expanded to the players specified in `group_members`.  |
```

Comment:
This service is implemented using `async_register_entity_service`, making `entity_id` optional.

```suggestion
| `entity_id`            |     yes | The media player entity whose playback will be expanded to the players specified in `group_members`.  |
```

Reply:
OK, thanks. I have updated the docs.